### ISSUE TITLE
Using an existing server

### ISSUE THREAD

**Issue by QzLP2P** [diogomartino/palworld-ds-gui#26]
Thanks you for the project, i have a question about existing server.

When we use existing server the PalworldDSGUI_Server.exe should be like this
```
SomeRandomFolder/
├── server/ <------- This is your existing server folder (the one with the **PalServer.exe** executable)
├── PalworldDSGUI_Server.exe
```

but the default folder is "PalServer" not server so when i launch the exe, it doesn't find any server.

It's possible to add some config to override the folder name ?

**Comment by diogomartino** [comment]
Hi, sure, I can add an override for that pretty easily, but any reason why you can't just rename `PalServer` to `server`?

**Comment by QzLP2P** [comment]
When i followed the tutorial the name was free ( so i decided to name it PalServer)
No Real reason for me to not rename it.
If I do it i have just to edit scheduled tasks.

i’m just asking if it’s possible with little effort, maybe than can help someone else :)

**Comment by diogomartino** [comment]
No problem, will release a new version either today or tomorrow and that will be included. Thanks for the feedback!

**Comment by diogomartino** [comment]
Done! Implemented in #27. Please read [the docs](https://github.com/diogomartino/palworld-ds-gui/blob/main/HOW_TO_USE.md#gui-server-parameters) for more info.
